docs/contribute/saros-testing-framework.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Saros Testing Framework (STF)
+---
+
+# {{ page.title }}
+{:.no_toc}
+
+{% include toc.html %}
+
+For further information
+about STF take a look at the [STF User
+Manual](http://saros-build.imp.fu-berlin.de/stf/STF_Manual.pdf).
+
+## Eclipse
+
+### Prerequisites
+
+* Development environment with **Eclipse 4.6**. It is also possible to use newer version, but some buttons are renamed in newer version which leads to a few error.
+
+### Configuration
+
+* Before you can run test tests you need to create accounts as described in the [user documentation](../../documentation/getting-started.html).
+* Then you have to create the file `configuration.properties` in directory `de.fu_berlin.inf.dpp/test/framework/stf/src/de/fu_berlin/inf/dpp/stf/client`.
+* Add the following lines to the config and **replace the placeholders with your credentials** (make sure **every tester has an unique JID**)
+
+```properties
+ALICE_JID = <insert user here>@saros-con.imp.fu-berlin.de/Saros
```

Comment:
This one was of the major goal when I added this configuration stuff. Yes you can use your OWN server. Otherwise you will have to create accounts, and depending on the configuration of the server and your possibility to change your IP address this may take some time.

Reply:
I Added the information and adjusted the property template